src/components/molecules/FirstBootomSection.js

```diff
@@ -0,0 +1,62 @@
+import React from "react"
+
+const FirstBootomSection = ({
+  title,
+  text,
+  buttonText,
+  buttonAction,
+}) => (
+  <div
+    style={{
+      height: 436,
```

Comment:
It would be ideal to have all of these styles live in [styled components](https://www.gatsbyjs.com/plugins/gatsby-plugin-styled-components/) rather than in-line